content/05kapitel.tex

```diff
@@ -320,6 +320,108 @@ \subsection{Figma Designs}
+    \includegraphics[width=\linewidth]{umlActivityFeed.png}
+    \caption{Aktivitätsdiagramm der Feedseite}
+    \label{fig:umlActivityFeed.png}
+\end{figure}
+
+Auf der Artikelseite kann wie in Abbildung~\ref{fig:umlActivityView.png} dargestellt der Artikel gelesen werden.
+Darüber hinaus kann der Artikel als Lesezeichen gespeichert, geliked und/oder geteilt werden.
+Auch die Feedseite des Artikels kann aufgerufen werden. Die Ursprungswebseite des Artikels kann auch geöffnet werden.
+Unter den ähnlichen Artikeln kann zu diesem per Klick gewechselt werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityView.png}
+    \caption{Aktivitätsdiagramm der Artikelseite}
+    \label{fig:umlActivityView.png}
+\end{figure}
+
+
+In Abbildung~\ref{fig:umlActivityBookmarks.png} ist das Aktivitätsdiagramm der Lesezeichenseite dargestellt.
+Alle vom Nutzer gespeicherten Lesezeichen werden hier aufgelistet und es kann gefiltert und/ oder darin gesucht werden.
+Einzelne Artikel können aus den Lesezeichen gelöscht werden. Auch können einzelne Artikel geliked oder entliked werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityBookmarks.png}
+    \caption{Aktivitätsdiagramm der gespeicherten Lesezeichenseite}
+    \label{fig:umlActivityBookmarks.png}
+\end{figure}
+
+Die Aktivitäten der Abonnementsseite werden in Abbildung~\ref{fig:umlActivitySubscription.png} abgebildet.
+Die Abonnements können gefiltert und/oder darin gesucht werden. Jeder abonnierte Feed kann abgewählt werden.
+Ein Feed kann ausgeklappt werden und einzelne Artikel geliked oder als Lesezeichen gespeichert werden.
+Die Artikel können mit einem Klick aufgerufen werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivitySubscription.png}
+    \caption{Aktivitätsdiagramm der Abonnements}
+    \label{fig:umlActivitySubscription.png}
+\end{figure}
+
+In der Abbildung~\ref{fig:umlActivityAddFeed.png} wird der Ablauf zum Hinzufügen eines neuen Feeds dargestellt.
+Ein RSS-Link wird eingetragen und mit klick auf "load" wird der Link geprüft. Ist der Link fehlerbehaftet muss er korrigiert werden.
+Funktioniert der Link werden Name und Beschreibung vorausgefüllt.
+Der Name und die Beschreibung kann vor dem finalen hinzufügen noch angepasst werden.
+Das Hinzufügen eines Feeds kann jederzeit über die Schaltfläche verlassen werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityAddFeed.png}
+    \caption{Aktivitätsdiagramm zum Hinzufügen eines Feeds}
+    \label{fig:umlActivityAddFeed.png}
+\end{figure}
+
+Der Nutzer kann die Einstellungen gemäß Abbildung~\ref{fig:umlActivitySettings.png} ändern.
+Es können die Accounteinstellungen, die Benachrichtigung und das Farbschema angepasst werden.
+Ausloggen kann sich der Nutzer in den Einstellungen auch.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivitySettings.png}
+    \caption{Aktivitätsdiagramm der Einstellungsseite}
+    \label{fig:umlActivitySettings.png}
+\end{figure}
```

Comment:
Change Account Settings führt zu Keycloak und nicht der Settings page (bitte als zwischenschritt adden)

Reply:
Und natürlich in text erläuten

Reply:
zwischenschritt  einfügen nach keycloak -> settingspage

Reply:
[x]

Reply:
Wo zwischenschritt?? 
![image](https://user-images.githubusercontent.com/54294591/167691820-88f9c4a5-68d7-4644-85c4-42a283d9be41.png)